tools/windows/DatadogAgentInstaller/WixSetup/Datadog Agent/AgentCustomActions.cs

```diff
@@ -178,6 +180,21 @@ public AgentCustomActions()
                 Impersonate = false
             };
 
+            EnsureGeneratedFilesRemoved = new CustomAction<CustomActions>(
+                new Id(nameof(EnsureGeneratedFilesRemoved)),
+                CustomActions.CleanupFiles,
+                Return.check,
+                When.Before,
+                Step.InstallFiles,
```

Comment:
We might want to do this before `DDCreateFolders` instead.

Reply:
On second thought this is fine. `DDCreateFolders` will ensure the permissions are set before we remove these files.